db/db_impl.h

```diff
@@ -545,7 +545,11 @@ class DBImpl : public DB {
   ColumnFamilyHandle* GetColumnFamilyHandle(uint32_t column_family_id);
 
   // Same as above, should called without mutex held and not on write thread.
-  ColumnFamilyHandle* GetColumnFamilyHandleUnlocked(uint32_t column_family_id);
+  std::unique_ptr<ColumnFamilyHandle> GetColumnFamilyHandleUnlocked(uint32_t column_family_id);
```

Comment:
I would like to remove `GetColumnFamilyHandleUnlocked` since only BlobDB use it but it don't even need to use it. However, the `GetColumnFamilyHandle` method has the same data race issue, and it would be great if we fix that too.

Reply:
Seems the implementation of `GetColumnFamilyHandle` is good because the caller should guarantee holding db mutex or under write thread.

TitanDB will use `GetColumnFamilyHandleUnlocked `, so maybe we can reserve it.

Reply:
ok, make sense.